demos/browser/server.js

```diff
@@ -145,6 +145,9 @@ function serve(host = '127.0.0.1:8080') {
         const languageCode = requestUrl.query.language;
         const region = requestUrl.query.region;
         let transcriptionConfiguration = {};
+        const transcriptionStreamParams = JSON.parse(requestUrl.query.transcriptionStreamParams);
```

Comment:
As we continue to add params, we should consider switching to use POST body rather than query params for everything.

Reply:
Looks like a good idea. Can club this with combining parameters in a single interface